topdown/http.go

```diff
@@ -85,11 +85,19 @@ func validateHTTPRequestOperand(term *ast.Term, pos int) (ast.Object, error) {
 
 }
 
+// Adds custom headers to a new HTTP request.
+func addHeaders(req *http.Request, headers map[string]interface{}) {
+	for k, v := range headers {
+		req.Header.Add(k, v.(string))
```

Comment:
This will also panic if the header value is not a string.  See comment below.

Reply:
Working on this one.